lib/text.rb

```diff
@@ -0,0 +1,23 @@
+require 'twilio-ruby'
+
+class Text
+
+  ACCOUNT_SID = 'ACc68fbb7c5f4b5a7c5bb1a06faaab901d'
+  AUTH_TOKEN = '4e5bdc58fcc22d2b586f396a43288263'
+
+  def initialize(number = XXX, client_number = XXX, object = Object)
```

Comment:
These injections are a little odd — perhaps you were trying to figure some things out?

If not — what's `XXX`? and what's the purpose of `object`?